### PR TITLE
remove shuffle filenames function

### PR DESCRIPTION
**Pull request by benjijamorris**
Porting to MonAI major changes

- Change supported models to Monai BasicUNet
- Update formatting to match pytorch lightning
- Change model checkpoint and loading functions to pytorch lightning callbacks
- Change dataloaders to a single dataloader with augmentation arguments in the config file
- Remove unused functions and imports
- Change model inference to monai sliding window inference
-

### CODE REVIEW

File: aicsmlsegment/bin/predict.py [hunk 1]
```diff
@@ -77,10 +77,12 @@ def main():
...
 
                 # apply the model
-                output_img = apply_on_image(model, img, args_inference)
+                output_img = apply_on_image(
+                    model, img, args_inference, squeeze=False, to_numpy=True
+                )
 
                 # extract the result and write the output
-                out = output_img[:, args_inference.OutputCh, :, :, :].cpu()
+                out = output_img[:, args_inference.OutputCh, :, :, :]
```
Comment (jxchen01) [inline aicsmlsegment/bin/predict.py:85]: do we need to convert it into cpu first?

Comment (jxchen01) [reply aicsmlsegment/bin/predict.py:85]: Ahaa, I see, the `to_numpy` option. I understand now

File: aicsmlsegment/model_utils.py [hunk 2]
```diff
@@ -3,10 +3,11 @@
 from monai.inferers import sliding_window_inference
 
 
-def apply_on_image(model, input_img, args):
+def apply_on_image(model, input_img, args, squeeze, to_numpy):
     """
     Perform inference on an input img through a model with or without runtime augmentation.
     If runtime augmentation is selected, perform inference on flipped images and average results.
+    returns: 5d np.array NCZYX order
```
Comment (jxchen01) [inline aicsmlsegment/model_utils.py]: can you add docstring for Parameters to give more details about each parameter here?

**Review by jxchen01 — APPROVED** [review]
the only suggestion is to add doc for apply_on_single_image(). Other than that, it looks super cool.

**Review by jxchen01 — APPROVED** [review]
